README.md

```diff
@@ -22,7 +22,7 @@
 The Kubernetes Cluster API Provider Hetzner enables declarative provisioning of multiple Kubernetes clusters on [Hetzner infrastructure](https://hetzner.cloud).
 High-available Kubernetes clusters on bare metal and on cloud instances are supported. The Cluster API simplifies the creation, updates, and operation of production-ready clusters. Self-managed Kubernetes clusters can, therefore, be easily operated even at a large scale.
 
-> Disclaimer: This is no official Hetzner project! It is maintained by the folks of the cloud-native start-up [Syself](https://syself.com).
+> Disclaimer: This is no official Hetzner project! It is maintained by [Syself](https://syself.com).
```

Comment:
Maybe would be better to say "This is not an official..." ?

Reply:
@buhodev yes, thank you "... not an ..." is better.